### DISCUSSION TITLE
What is the best way to provide review challenges back to the team?

### DISCUSSION THREAD

**Discussion by toddpi314** [pixeleye-io/pixeleye#337] [Q&A]
Ideally, there would either be a slack or github Pull Request integration so the screenshot diffs can somehow surface as a quality challenge for the team.

**Comment by AlfieJones** [comment]
Pixeleye should surface any review challenges as an error in the github integration (if importing a project via a github social login).

A slack integration would be awesome though! I'll add it to my list of things to investigate

**Comment by ghost** [comment]
Generic webhooks would also be nice for generic VSC/PR
